### PR TITLE
Add docs for volume expansion and online resizing

### PR DESCRIPTION
**Pull request by gnufied**
Add documentation for volume expansion going beta and for volume online resizing.

xref https://github.com/kubernetes/features/issues/531
xref https://github.com/kubernetes/features/issues/284

### CODE REVIEW

**Comment by mistyhacks** [comment]
@gnufied I just added a copyedit commit. PTAL and lgtm if the meaning is still correct.

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -196,17 +197,33 @@ parameters:
 allowVolumeExpansion: true
 ```
 
-Once both feature gate and the aforementioned admission plug-in are turned on, a user can request larger volume for their `PersistentVolumeClaim`
-by simply editing the claim and requesting a larger size.  This in turn will trigger expansion of the volume that is backing the underlying `PersistentVolume`.
+To request a larger volume for a PVC, edit the PVC's configuration and request a larger
```
Comment (gnufied) [inline content/en/docs/concepts/storage/persistent-volumes.md]: I am not sure I would use wording "edit the PVC's configuration" to refer editing the pvc's yaml - I was thinking something like "edit PVC object" would be better?

**Comment by mistyhacks** [comment]
Updated, PTAL

/approve

**Comment by lucaim** [comment]
Sorry to bother @mistyhacks , @gnufied : should _PersistentVolumeClaimResize_ admission plugin be enabled? Every reference disappeared from the documentation.

**Comment by gnufied** [comment]
@lucaim yes that is enabled by default and there is no need to enable it manually. The admission controller page should still list the controller

**Pull request merged**